src/prefect/tasks/databricks/databricks_submitjob.py

```diff
@@ -686,3 +786,335 @@ def run(
+            "tasks": [
+                {
+                    "task_key": "Sessionize",
+                    "description": "Extracts session data from events",
+                    "depends_on": [],
+                    "existing_cluster_id": "0923-164208-meows279",
+                    "spark_jar_task": {
+                        "main_class_name": "com.databricks.Sessionize",
+                        "parameters": ["--data", "dbfs:/path/to/data.json"],
+                    },
+                    "libraries": [{"jar": "dbfs:/mnt/databricks/Sessionize.jar"}],
+                    "timeout_seconds": 86400,
+                },
+                {
+                    "task_key": "Orders_Ingest",
+                    "description": "Ingests order data",
+                    "depends_on": [],
+                    "existing_cluster_id": "0923-164208-meows279",
+                    "spark_jar_task": {
+                        "main_class_name": "com.databricks.OrdersIngest",
+                        "parameters": ["--data", "dbfs:/path/to/order-data.json"],
+                    },
+                    "libraries": [{"jar": "dbfs:/mnt/databricks/OrderIngest.jar"}],
+                    "timeout_seconds": 86400,
+                },
+                {
+                    "task_key": "Match",
+                    "description": "Matches orders with user sessions",
+                    "depends_on": [
+                        {"task_key": "Orders_Ingest"},
+                        {"task_key": "Sessionize"},
+                    ],
+                    "new_cluster": {
+                        "spark_version": "7.3.x-scala2.12",
+                        "node_type_id": "i3.xlarge",
+                        "spark_conf": {"spark.speculation": True},
+                        "aws_attributes": {
+                            "availability": "SPOT",
+                            "zone_id": "us-west-2a",
+                        },
+                        "autoscale": {"min_workers": 2, "max_workers": 16},
+                    },
+                    "notebook_task": {
+                        "notebook_path": "/Users/user.name@databricks.com/Match",
+                        "base_parameters": {"name": "John Doe", "age": "35"},
+                    },
+                    "timeout_seconds": 86400,
+                },
+            ],
+            "run_name": "A multitask job run",
+            "timeout_seconds": 86400,
+            "idempotency_token": "8f018174-4792-40d5-bcbc-3e6a527352c8",
+            "access_control_list": [
+                {
+                    "user_name": "jsmith@example.com",
+                    "permission_level": "CAN_MANAGE",
+                }
+            ],
+        })
+
+        with Flow("my flow") as f:
+            conn = PrefectSecret('DATABRICKS_CONNECTION_STRING')
+            submit_multitask_run(**databricks_kwargs, databricks_conn_secret=conn)
```

Comment:
`submit_multitask_run` isn't defined in this example and it's a bit different from the first example so there is some overhead